### PR TITLE
Update metadata to reflect newest maintainer

### LINKED ISSUE #2749: Update maintainers for vsphere-cpi
I believe the maintainers list for the vsphere-cpi is out of date. This should be updated:

https://github.com/vmware-tanzu/community-edition/blob/5b8537e2c64b902553b57956af0ba650b18b2387/addons/packages/vsphere-cpi/metadata.yaml#L10-L13

Perhaps it should be sync'd with this member list?
https://github.com/orgs/vmware-tanzu/teams/pkg-vsphere-cpi-owners/members

Not sure if that list is up to date, but it looks closer to me...

cc @vmware-tanzu/pkg-vsphere-cpi-owners

### PR DESCRIPTION
**Pull request by lubronzhan**
## What this PR does / why we need it

CPI Maintainer list in metadata is outdated

## Details for the Release Notes (PLEASE PROVIDE)

```release-note
Update the CPI maintainer list in metadata file
```

## Which issue(s) this PR fixes

Fixes: #https://github.com/vmware-tanzu/community-edition/issues/2749

### CODE REVIEW

**Comment by lubronzhan** [comment]
cc @vmware-tanzu/pkg-vsphere-cpi-owners

**Comment by stmcginnis** [comment]
cc @seemiller @jpmcb

File: addons/packages/vsphere-cpi/metadata.yaml [hunk 1]
```diff
@@ -8,8 +8,9 @@ spec:
   shortDescription: "vSphere Cloud Provider Interface"
   providerName: VMware
   maintainers:
-    - name: Andrew Sy Kim
-    - name: Lucheng Bao
-    - name: Nicole Han
+    - name: Lubron Zhan
+    - name: John McBride
+    - name: Nicholas Seemiller
```
Comment (jpmcb) [inline addons/packages/vsphere-cpi/metadata.yaml:13]: I'm not sure @seemiller and I should be added as maintainers explicitly in this package.

We are only apart of the @vmware-tanzu/pkg-vsphere-csi-owners group for admin operations.

Comment (lubronzhan) [reply addons/packages/vsphere-cpi/metadata.yaml:13]: IC. let me remove

**Review by jpmcb — APPROVED** [review]

**Pull request merged**